### PR TITLE
Split envoy config test

### PR DESCRIPTION
**Pull request by kyessenov**

### CODE REVIEW

File: proxy/envoy/config_test.sh [hunk 1]
```diff
@@ -2,21 +2,20 @@
...
-for f in ${CONFIG_FILES}; do
-  echo "##############################"
-  echo "# Testing $f..."
-  echo "##############################"
-  ${ENVOY} -c proxy/envoy/testdata/$f.json.golden &
-  PID=$!
-  sleep 1
-  kill -KILL $PID
-done
+echo "##############################"
```
Comment (ayj) [inline proxy/envoy/config_test.sh]: I don't think these three echo lines are needed anymore?

File: proxy/envoy/config_test.go [hunk 2]
```diff
@@ -312,6 +314,7 @@ func makeMeshConfig() proxyconfig.ProxyMeshConfig {
 	mesh := DefaultMeshConfig
 	mesh.MixerAddress = "localhost:9091"
 	mesh.DiscoveryAddress = "localhost:8080"
+	mesh.DiscoveryRefreshDelay = &duration.Duration{Nanos: int32(10 * time.Millisecond / time.Nanosecond)}
```
Comment (ayj) [inline proxy/envoy/config_test.go]: You can use the ptypes package to help here, e.g.

```golang
import "github.com/golang/protobuf/ptypes/ptypes"

mesh.DiscoveryRefreshDelay = ptypes.DurationProto(10 * time.Millisecond)
```

File: proxy/envoy/BUILD [hunk 3]
```diff
@@ -59,8 +59,65 @@ genrule(
 )
 
 sh_test(
-    name = "config_test",
+    name = "envoy_config_v0_test",
```
Comment (ayj) [inline proxy/envoy/BUILD]: Do these need a dependency on `envoy_bin` or does bazel figure that out via `data` attribute?

Comment (kyessenov) [inline proxy/envoy/BUILD]: "envoy" data dependency is sufficient for genrules.

File: proxy/envoy/config_test.go (hunk 2 above)
Comment (kyessenov) [inline proxy/envoy/config_test.go]: ack, that's better.

File: proxy/envoy/config_test.sh (hunk 1 above)
Comment (kyessenov) [inline proxy/envoy/config_test.sh]: removing them

**Pull request merged**